volume/profiles/profiles.go

```diff
@@ -310,8 +285,8 @@ func (pp *pvcVolProProfile) StorageSize() (string, error) {
 }
 
 // ReplicaCount get the number of replicas required
-func (pp *pvcVolProProfile) ReplicaCount() (*int32, error) {
-	specs := pp.pvc.Specs
+func (pp *defVolProProfile) ReplicaCount() (*int32, error) {
```

Comment:
Similarly for other (for example: GetReplicaCount) methods too. Errors should be handled and displayed in the logs, so that it'll help while troubleshooting.